### PR TITLE
Deprecating Activity Finder 3.x branch. We have it in 4.x branch for upgrade needs

### PR DESCRIPTION
**Pull request by podarok**
Original Issue, this PR is going to fix: https://openy.atlassian.net/browse/MAINTAIN-74

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [ ] Ensure there is no v3 available in composer.json.
- [ ] Check docs update in AF https://github.com/ymcatwincities/openy_activity_finder/pull/60

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3181/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3181/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3181/

File: composer.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
         "symfony/dom-crawler": "~2.8 || ~3.0 || ~4.2",
         "symfony/process": "^4",
         "ymcatwincities/media_entity_document": "*",
-        "ymcatwincities/openy_activity_finder": "^3.1 || ^4.0",
+        "ymcatwincities/openy_activity_finder": "^4.0",
```
Comment (anpolimus) [inline composer.json:114]: @podarok how can we work in a situation if a new D9 project wants AF3?

Comment (podarok) [reply composer.json:114]: I guess this is solved in Slack https://openy.slack.com/archives/C6G4ZTLVC/p1621426840002300
4.0 releases of AF contains AFv3 UI components for backward compatibility

**Review by hamrant — APPROVED** [review]

**Review by anpolimus — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3229/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3229/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3229/

**Pull request merged**